python-package/xgboost/spark/data.py

```diff
@@ -228,6 +260,10 @@ def append_dqm(part: pd.DataFrame, name: str, is_valid: bool) -> None:
 
     def make(values: Dict[str, List[np.ndarray]], kwargs: Dict[str, Any]) -> DMatrix:
         if len(values) == 0:
+            get_logger("XGBoostPySpark").warning(
```

Comment:
previously, the warning is only printed for empty training data. while this PR also prints it for validation data.

Reply:
It's still good to have a warning, the support is "best effort" and is not yet extensively tested. If something goes wrong at least users will have some clues to debug.